composer.json

```diff
@@ -27,6 +27,7 @@
     "php-stubs/wordpress-stubs": "^v5.9.0",
     "phpcompatibility/phpcompatibility-wp": "^2.1",
     "phpmd/phpmd": "^2.9",
+    "phpstan/phpstan": "^1.4",
```

Comment:
Note that here's a dedicated ticket #4817 to change the PHPStan dependency.

There, we don't need to include `phpstan/phpstan` as we only need to require `php-stubs/wordpress-stubs`

I would like to keep this change separate.

Reply:
I don't see the harm in doing this here. If anything it will speed composer install up and making upgrading easier.